frontend/components/LabeledBtn.vue

```diff
@@ -17,7 +18,7 @@
       'text-base sm:text-lg xl:text-3xl xl:px-6 xl:py-3': fontSize == '3xl',
     }"
   >
-    {{ $t(label) }}
+    {{ buttonText }}
   </a>
   <NuxtLink
```

Comment:
This `<NuxtLink>` tag should be given an `aria-label` attribute that has `alternateText` assigned to it 😊